resources/shaders/glbillbshader.frag

```diff
@@ -1,9 +1,10 @@
-#version 410 core
+precision highp float;
+precision highp usamplerBuffer;
```

Comment:
`precision highp usamplerBuffer; ` is open gl 4.4+ . This may cause shader building errors on some 4.1 systems. Will discuss on discord.